MerlinAU.sh

```diff
@@ -4408,15 +4435,12 @@ _GetLoginCredentials_()
         printf "Encoded Credentials:\n"
         printf "${GRNct}$loginCredsENC${NOct}\n"
 
-        # Prompt to test the credentials #
         if _WaitForYESorNO_ "\nWould you like to test the current login credentials?"
         then
             _TestLoginCredentials_ "$loginCredsENC" || continue
-        else
-            _UpdateLoginPswdCheckHelper_ UNKNOWN
```

Comment:
This call to set the password status to UNKNOWN is unnecessary. 
The goal is to leave the previously check status alone. If it was already verified and is still the same, we don't want to set it to UNKNOWN just because the user chooses not to run the password check again.
Also, the function checks for that scenario, so no need to make the call to UNKNOWN.